app/models/feeds/apple_subscription.rb

```diff
@@ -31,6 +33,10 @@ def self.model_name
     Feed.model_name
   end
 
+  def publish_feeds
+    podcast&.publish!
```

Comment:
As per suggestion, consolidating the "getting ready to publish to Apple" sort of logic in this class.

I wish I could just trigger a publish of the feed without even trying to publish to apple when this first gets created, and before we want to try and sync with apple...maybe I will add a way to do that, but not sure how to decouple those in the current job?